docs/installation.md

```diff
@@ -25,3 +25,24 @@ Dev and production ready assets are available in the [`dist` folder of the repos
 
 - Dev version, including source maps: [kinto.dev.js](https://cdn.rawgit.com/Kinto/kinto.js/1.0.0-rc.1/dist/kinto.dev.js)
 - Production version, minified, no source maps: [kinto.dev.js](https://cdn.rawgit.com/Kinto/kinto.js/1.0.0-rc.1/dist/kinto.min.js)
+
+
+### Subresource integrity
+
+To make sure that you are using the right code when loading from a CDN, you can use subresource
+integrity with the hash provided in ``kinto.min.sha384`` with your release.
```

Comment:
We could add something like "You can see the compatible browsers at http://caniuse.com/#search=SRI". I'm not sure why it doesn't show up there. @mozfreddyb any idea?